CVC5 build failure on Windows

Example from [this build log](https://github.com/GaloisInc/what4-solvers/actions/runs/5094776099/jobs/9158987946?pr=35#step:10:337):

```
In file included from D:/a/what4-solvers/what4-solvers/repos/cvc5/include/cvc5/cvc5.h:21,
                 from D:/a/what4-solvers/what4-solvers/repos/cvc5/src/api/cpp/cvc5.cpp:33:
D:/a/what4-solvers/what4-solvers/repos/cvc5/include/cvc5/cvc5_kind.h:43:11: error: found ':' in nested-name-specifier, expected '::'
   43 | enum Kind : int32_t
      |           ^
      |           ::
D:/a/what4-solvers/what4-solvers/repos/cvc5/include/cvc5/cvc5_kind.h:43:6: error: 'Kind' has not been declared
   43 | enum Kind : int32_t
      |      ^~~~
D:/a/what4-solvers/what4-solvers/repos/cvc5/include/cvc5/cvc5_kind.h:44:1: error: expected unqualified-id before '{' token
   44 | {
      | ^
D:/a/what4-solvers/what4-solvers/repos/cvc5/include/cvc5/cvc5_kind.h:5617:26: error: 'Kind' was not declared in this scope
 5617 | std::string kindToString(Kind k) CVC5_EXPORT;
      |                          ^~~~
```

I'm not entirely sure why this happens yet. Perhaps the C compiler isn't using C++11 or later for some reason? I'm not sure why else `enum Kind : int32_t` would fail to parse.

--- a/.github/ci.sh
+++ b/.github/ci.sh
@@ -88,6 +88,8 @@ build_cvc4() {
 
 build_cvc5() {
   pushd repos/cvc5
+  # Work around https://github.com/cvc5/cvc5/issues/9778
+  patch -p1 -i $PATCHES/cvc5-gcc-13-fix.patch
   if $IS_WIN ; then
     # Work around https://github.com/cvc5/cvc5/issues/9564
     patch -p1 -i $PATCHES/cvc5-win64-native.patch

--- a/patches/cvc5-gcc-13-fix.patch
+++ b/patches/cvc5-gcc-13-fix.patch
@@ -1,0 +1,12 @@
+diff --git a/include/cvc5/cvc5_kind.h b/include/cvc5/cvc5_kind.h
+index 0ad198b55..64098e18a 100644
+--- a/include/cvc5/cvc5_kind.h
++++ b/include/cvc5/cvc5_kind.h
+@@ -18,6 +18,7 @@
+ #ifndef CVC5__API__CVC5_KIND_H
+ #define CVC5__API__CVC5_KIND_H
+ 
++#include <cstdint>
+ #include <ostream>
+ 
+ namespace cvc5 {

